vmdb/lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -131,6 +131,7 @@ class AutomateMethodException < StandardError
   Socket.do_not_reverse_lookup = true  # turn off reverse DNS resolution
 
   require 'drb'
+  require 'yaml'
```

Comment:
It doesn't surprise me; relying on what AS might internally require as its own dependency is a bug.